### ISSUE TITLE
[Feature request] Add Bookmarks to tree view

### ISSUE THREAD

**Issue by aevans0001** [lxqt/pcmanfm-qt#1803]
**Is your feature request related to a problem? Please describe.**
I dislike having to click a menu to see bookmarks, when they could simply be in list view.
In places view i hate having to click on the directory to see subfolders and then have to move to the right to click on subdirectories

**Describe the solution you'd like**
Add tree view to lists view or add a quicklaunch bar (like windows) to tree view.  I would also like to see network and other subcategories

**Comment by tsujan** [comment]
> I dislike having... i hate having...

The list and directory tree views are complementary by design and can be switched just by turning the mouse wheel on the upper combo box. I'm afraid your negative emotions aren't good reason for changing that.

One can also say that a GUI which mixes them together would be too cluttered (like that of Dolphin).

Anyway, we accept patches. If you have a clean code which mixes them *optionally*, please make a PR.

**Comment by ahsand97** [comment]
Something like this? @tsujan what do you think?

<img width="3840" height="2102" alt="Image" src="https://github.com/user-attachments/assets/825ba92c-4bbc-4e03-8c53-c3085ab08b70" />

**Comment by tsujan** [comment]
> Something like this?

Oh, no! That would be a mess ;) It contradicts the design too (read above).

Personally, I think this report should be closed. People think in terms of Dolphin or some other FM and expect pcmanfm-qt should be designed similarly. Well, it isn't.

**Comment by ahsand97** [comment]
> > Something like this?
> 
> Oh, no! That would be a mess ;) It contradicts the design too (read above).
> 
> Personally, I think this report should be closed. People think in terms of Dolphin or some other FM and expect pcmanfm-qt should be designed similarly. Well, it isn't.

I totally agree, I was just experimenting and it didn't look quite right haha.
